### ISSUE TITLE
safari ad blocker checkout doesn't load - GA enabled

### ISSUE THREAD

**Issue by asim-vax** [magento/magento2#21326]
### Preconditions (*)
1. php7.0-7.2
2. mysql
3. magento 2.2.6 & magento 2.3

### Steps to reproduce (*)
1. vanilla install
2. create test product
3. enable GA in stores -> config -> sales -> Google API
4. use safari
5. add test product to basket
6. enable ublock origin, visit checkout
7. infinite loader
8. disable ublock origin, reload page
9. checkout works

This was fixed for majority of browsers in #12428

### Expected result (*)
1. add product to basket
2. checkout loads

### Actual result (*)
1. add product to basket
2. infinite loader, checkout never finishes loading

**Comment by magento-engcom-team** [comment]
Hi @asim-vax. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me 2.3-develop instance` - upcoming 2.3.x release

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@asim-vax do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by asim-vax** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @asim-vax. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @asim-vax, here is your [Magento instance](https://i-21326-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-21326-2-3-develop.instances.magento-community.engineering/admin](https://i-21326-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

-  _Next steps are available in case you are a member of **Community Maintainers**._

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by asim-vax** [comment]
I can confirm i can reproduce this issue on the 2.3 vanilla instance generated

**Comment by asim-vax** [comment]
Also, as a sidenote - your SSL cert is invalid, meaning some browsers reject it

**Comment by asim-vax** [comment]
@engcom-backlog-nazar have you been able to replicate this? As it should be fairly easy to replicate

**Comment by ghost** [comment]
@asim-vax yes this is very easy to replicate on macbook :) but i'm not have an safari browser

**Comment by ghost** [comment]
@asim-vax maybe on next 2 days i will be able to recheck on safari.

**Comment by asim-vax** [comment]
@engcom-backlog-nazar has there been any update?

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-backlog-nazar 
 Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-98686`, `MAGETWO-98687` were created

**Issue Available**: @engcom-backlog-nazar, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by ladle3000** [comment]
Has this been fixed yet? Some customers with outdated Macs are unable to checkout for us as well. Infinite loader. Also I'm running 2.3.0

**Comment by engcom-Bravo** [comment]
Hi,

Thanks for your reporting and collaboration.

We have tried to reproduce the issue in latest 2.4-develop instance and the issue is no more reproducible.kindly refer the screenshots.

<img width="1266" alt="Image" src="https://github.com/user-attachments/assets/8b040c14-f5fb-4437-b519-e8855f781c55" />

checkout loads successfully.

Hence we are closing this issue.

Thanks.
